src/InputGuesser.tsx

```diff
@@ -34,13 +34,28 @@ import type {
   IntrospectedInputGuesserProps,
 } from './types.js';
 
+// Transform enum value to humanize representation.
+export const defaultTransformEnum = (value: string | number): string | number =>
+  // Camel case/ schema.org type to sentence.
+  `${value}`
```

Comment:
This behavior is very specific: I prefer if the user defines it manually instead of using this regexp which can cause some issues.